frontend/packages/knative-plugin/src/utils/__tests__/create-eventsources-utils.spec.ts

```diff
@@ -21,15 +21,15 @@ describe('Create knative Utils', () => {
     );
   });
 
-  it('expect response to schedule in spec for CronJobSource', () => {
-    const defaultEventingData = getDefaultEventingData(EventSources.CronJobSource);
+  it('expect response to data and schedule in spec for CronJobSource', () => {
```

Comment:
@nemesis09 why is `it` block description changed?

Reply:
reverted back to original description